iwyu.cc

```diff
@@ -1601,12 +1601,10 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
     // the namespace of the decl.  If we have 'using std::vector;' +
     // 'std::vector<int> foo;' we don't actually care about the
     // using-decl.
-    // TODO(csilvers): maybe just insert our own using declaration
-    // instead?  We can call it "Use what you use". :-)
```

Comment:
Yes, it does seem a little pretentious.

On the other hand, I think IWYU already has this behavior wrt `#include`. When run on `a.cc`, it modifies the associated header `a.h` in ways that might break consumers of the header (e.g. removing unused includes that were exported and used downstream.)

I'm not necessarily arguing for keeping the comment, I'm just thinking out loud... :-)